### PR TITLE
Main Image Component Review

### PR DESCRIPTION
**Pull request by WildMountainAir**
user-facing flow:
Index.jsx -> App.jsx -> titleImage.jsx & mainImage.jsx

### CODE REVIEW

File: client/dist/styles.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/********************
```
Comment (m0nstera) [inline client/dist/styles.css:1]: Love the organization of your stylesheet!

File: client/lib/routes.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    "id": 1,
+    "name": "Camo Onesie",
+    "slogan": "Blend in to your crowd",
+    "description": "The So Fatigues will wake you up and fit you in. This high energy camo...",
+    "category": "Jackets",
+    "default_price": "140"
+  }
+]
+*/
+const listProducts = (callback) => {
```
Comment (m0nstera) [inline client/lib/routes.js:25]: really clear code structure

File: client/src/app.jsx [hunk 3]
```diff
@@ -1,39 +1,57 @@
 import React from 'react';
-import Container from 'react-bootstrap/Container'
-import Jumbotron from 'react-bootstrap/Jumbotron'
-import Button from 'react-bootstrap/Button';
-import Col from 'react-bootstrap/Col';
-import Row from 'react-bootstrap/Row';
+import { Container, Row } from 'react-bootstrap';
```
Comment (m0nstera) [inline client/src/app.jsx:2]: nice destructuring :)

File: client/src/app.jsx [hunk 4]
```diff
@@ -1,39 +1,57 @@
...
+  //     throw(error)
+  //   } else {
+  //     console.log(data.results);
+  //   }
+  // })
+
+  render() {
+    return (
+      <Container>
+        <br />
```
Comment (m0nstera) [inline client/src/app.jsx:26]: I was wondering how to space out components; padding/margin/br. Do you often use <br>s? Purely inquisitive!

File: client/src/components/titleImage.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```
Comment (m0nstera) [inline client/src/components/titleImage.jsx:1]: organized and succinct component

**Review by m0nstera — CHANGES_REQUESTED** [review]

File: client/src/app.jsx (hunk 4 above)
Comment (WildMountainAir) [reply client/src/app.jsx:26]: I do use this often, usually with divs but I've been experimenting with Container a ton

**Pull request merged**